Structure_Engine/Create/Physical/Wall.cs

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+using System.Collections.Generic;
+using System.Linq;
+using BH.oM.Structure.Elements;
+using BH.oM.Geometry;
+using BH.oM.Base.Attributes;
+using System.ComponentModel;
+using BHPE = BH.oM.Physical.Elements;
+using BHPC = BH.oM.Physical.Constructions;
+using BH.oM.Physical.Reinforcement;
+using BH.oM.Structure.SurfaceProperties;
+using BH.oM.Structure.Reinforcement;
+using BH.oM.Structure.Fragments;
+using BH.Engine.Base;
+using BH.oM.Quantities.Attributes;
+
+namespace BH.Engine.Structure
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+        [Description("Creates a physical Wall from a Panel. The Wall will be assigned a Construction based on the SurfaceProperty of the Panel.")]
+        [Input("panel", "The Panel to use as the base for the framing element.")]
+        [Output("wall", "The created physical Wall based on the Panel element provided.")]
+        public static BHPE.Wall Wall(Panel panel)
```

Comment:
This could possibly become a generic method, taken that it is exactly the same as the Floor creation one? But not 200% sure, so feel free to ignore

Reply:
I meant you could try writing a method like `Create.Surface<T>(Panel panel) where T : ISurface`, and depending on type of T you create wall, floor etc.

Reply:
Tried this, and had some issues with how it came up in Grasshopper- I'd prefer to leave this to a later date, perhaps when more panel types are implemented in Physical oM

Reply:
Sounds good to me - just in case, my idea was that we would have methods like `Create.Wall` and `Create.Floor` that would have only one line calling a private generic method `Create.ISurface<T>`. So from the UI perspective everything should stay the same. But happy for you to leave as is, actually I am not 100% sure whether my suggested change would be sustainable (thinking of minor differences in how floor or wall gets created)